projects/cdk/date-time/day.ts

```diff
@@ -122,17 +123,45 @@ export class TuiDay extends TuiMonth {
         );
     }
 
+    static parseRawDateString(
+        date: string,
+        dateMode: TuiDateMode = 'DMY',
+    ): {day: number; month: number; year: number} {
```

Comment:
nit: maybe add assertion for invalid string?

Reply:
done